magpie-submission-convert

```diff
@@ -79,3 +79,26 @@ then
 	export MAGPIE_TIMELIMIT_MINUTES=$(expr ${PBS_WALLTIME} / 60)
     fi
 fi
+
+if [ "${MAGPIE_SUBMISSION_TYPE}" == "lsf" ] || [ "${MAGPIE_SUBMISSION_TYPE}" == "lsfmpirun" ]
```

Comment:
no need to check for "lsf" here, since there's no need for backwards compatibility.  We only support "lsfmpirun"